### DISCUSSION TITLE
Does Transformer Variant Group work with PostCSS?

### DISCUSSION THREAD

**Discussion by john-prutton** [unocss/unocss#2895] [Q&A]
Hi, Im working in Nextjs and have set up UnoCSS to work with PostCSS via the PostCSS plugin  @unocss/postcss. Everything has been working fine. But when I follow the documentation to add transformerVariantGroup() to my transformers in uno.config.ts, the transformer does not function correctly.

Is this a known caveat of working with PostCSS? Am I doing something wrong, even after following the [docs](https://unocss.dev/transformers/variant-group) a tee? Please help. 😅

**Comment by p7gg** [comment]
same question
